packages/utils/src/types/actions.ts

```diff
@@ -0,0 +1,150 @@
+import { type TransactionRequest } from 'viem'
+import type {
+  FilterOperator,
+  TransactionFilter,
+  PluginActionNotImplementedError,
+} from '@rabbitholegg/questdk'
+import { type Address } from 'viem'
+import type { MintIntentParams } from './intents'
+
+export type SwapActionParams = {
```

Comment:
might be missing something but why is this duplicated into plugins?  Looking at sibling PR for the questdk, and these types were not removed?